app/controllers/TagManagementApi.scala

```diff
@@ -208,4 +208,10 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     Ok(Json.toJson(jobs))
   }
 
+  def deleteJob(id: Long) = (APIAuthAction andThen DeleteJobPermissionsCheck) {
+    JobRepository.deleteJob(id)
```

Comment:
Do we want to deal with `Dynamo.jobTable.deleteItem` throwing exceptions? Or just let play framework deal with it as a 500?